api/Models.py

```diff
@@ -52,6 +52,11 @@ def find_all(self):
             listing["_id"] = str(listing['_id'])
         return listings
 
+    def find_user_listings(self, ownerId):
+        listings = list(self.collection.find({ "owner" : ownerId}))
+        for listing in listings:
+            listing["_id"] = str(listing['_id'])
+        return listings
```

Comment:
clean use of find()! I need to implement this for my messages find_all() function

Reply:
Thanks!